openrtb2/channel.go

```diff
@@ -0,0 +1,49 @@
+package openrtb2
+
+import "encoding/json"
+
+// 3.2.24 Object: Channel
+//
+// This object describes the channel an ad will be displayed on.
+// A Channel is defined as the entity that curates a content library, or stream within a brand name for viewers.
+// Examples are specific view selectable ‘channels’ within linear and streaming television (MTV, HGTV, CNN, BBC One, etc) or a specific stream of audio content commonly called ‘stations.’
+// Name is a human-readable field while domain and id can be used for reporting and targeting purposes.
+// See 7.6 for further examples.
+type Channel struct {
```

Comment:
TODO: also adcom1, it seems

Reply:
Screw it as well, adcom1 vs openrtb2 objects are not always compatible - address in next major version or simply ignore.